website/www/site/content/en/contribute/release-guide.md

```diff
@@ -1126,21 +1126,28 @@ Verify that:
 * Images with *latest* tag are pointing to current release by confirming 
   1. Digest of the image with *latest* tag is the same as the one with {RELEASE} tag.
 
+### Merge website pull request
+
+Merge the website pull request to [list the release](/get-started/downloads/), publish the [Python API reference manual](https://beam.apache.org/releases/pydoc/), the [Java API reference manual](https://beam.apache.org/releases/javadoc/) and Blogpost created earlier.
+
 ### Git tag
 
 Create and push a new signed tag for the released version by copying the tag for the final release candidate, as follows:
 
-    VERSION_TAG="v${RELEASE}"
-    git tag -s "$VERSION_TAG" "$RC_TAG"
-    git push upstream "$VERSION_TAG"
+```
+VERSION_TAG="v${RELEASE}"
+git tag -s "$VERSION_TAG" "$RC_TAG"
+git push origin "$VERSION_TAG"
```

Comment:
The remote name depends on the users's `git remote` settings, which are arbitrary. I think we should be able to use `https://github.com/apache/beam` instead.

Reply:
Yeah, I just figured that `origin` is closer to a standard than `upstream`. That's a great point that we can just use the full URL though

Reply:
Done